lib/KeyboardAwareHOC.js

```diff
@@ -491,20 +491,22 @@ function KeyboardAwareHOC(
                 (error: ?string, fieldX: number, fieldY: number, fieldWidth: number, fieldHeight: number, 
                   caretX: number, caretY: number, caretRelativeX: number, caretRelativeY: number, 
                   caretWidth: number, caretHeight: number, textInputBottomTextInset: number ) => {
+                    const { height: windowHeight } = Dimensions.get( 'window' );
```

Comment:
using window height might fail in cases where the scrollable view has smaller height. like if we put something above the editor and editors height becomes smaller. for that reason I remember populating "topDistanceToWindow", "bottomDistanceToWindow" type of variables.